Gemfile

```diff
@@ -27,11 +27,15 @@ gem 'turbolinks', '~> 5'
 gem 'jbuilder', '~> 2.5'
 # Use Redis adapter to run Action Cable in production
 # gem 'redis', '~> 3.0'
+
+
 # Use ActiveModel has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
-
+gem 'carrierwave', '~> 1.0'
```

Comment:
The latest version from April is `1.1.0` but `1.0.0` is currently locked due to the version resolution. I recommend using a more pessimistic version pin:

```ruby
gem 'carrierwave', '~> 1.1.0'
```